### PR TITLE
[DEM-1205] Issues with instrument adapters

### PR DESCRIPTION
**Pull request by qSaevar**

### CODE REVIEW

**Comment by CumulonimbusCalvus** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/qtt/qtt/pullRequest?prid=4532354&bid=15255531)

File: src/qtt/instrument_drivers/adapters/virtual_dac_instrument_adapter.py [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
+from qtt.instrument_drivers.gates import VirtualDAC
 
 
-class VirtualDACInstrumentAdapter(SpiModuleInstrumentAdapter):
+class VirtualDACInstrumentAdapter(InstrumentAdapter):
     """ Adapter for the qtt VirtualDAC."""
 
-    def __init__(self, address: str) -> None:
-        super().__init__(address)
+    def __init__(self, address: str, instrument_name: Optional[str] = None) -> None:
```
Comment (CumulonimbusCalvus) [inline src/qtt/instrument_drivers/adapters/virtual_dac_instrument_adapter.py]: Codacy Issue found: [invalid syntax](https://app.codacy.com/app/CumulonimbusCalvus/qtt/pullRequest?prid=4532354)

File: src/qtt/instrument_drivers/adapters/time_stamp_instrument_adapter.py [hunk 2]
```diff
@@ -1,25 +1,14 @@
...
+from qilib.configuration_helper.adapters import CommonInstrumentAdapter
+from qilib.utils import PythonJsonStructure
 from qtt.instrument_drivers.TimeStamp import TimeStampInstrument
 
 
-class TimeStampInstrumentAdapter(InstrumentAdapter):
-    def __init__(self, address: str) -> None:
-        super().__init__(address)
+class TimeStampInstrumentAdapter(CommonInstrumentAdapter):
+    def __init__(self, address: str, instrument_name: Optional[str] = None) -> None:
```
Comment (CumulonimbusCalvus) [inline src/qtt/instrument_drivers/adapters/time_stamp_instrument_adapter.py]: Codacy Issue found: [invalid syntax](https://app.codacy.com/app/CumulonimbusCalvus/qtt/pullRequest?prid=4532354)

File: src/qtt/instrument_drivers/adapters/virtual_dac_instrument_adapter.py (hunk 1 above)
Comment (qSaevar) [reply src/qtt/instrument_drivers/adapters/virtual_dac_instrument_adapter.py]: Do you know why this is invalid syntax? Should I remove the typehints?

**Pull request merged**